### PR TITLE
Improving Login screen design

### PR DESCRIPTION
**Pull request by znegrin**
This is based on the issue [#ENG-650](https://athenianco.atlassian.net/browse/ENG-650). It adds a new design for the login screen. I took the freedom to adapt the existing template and also to modify all the states to make them fit on the new template design. 

Please, @dpordomingo if you think that I need to modify something, just let me know.

image

Signed-off-by: Zuri Negrín <zurinegrin@gmail.com>

### CODE REVIEW

File: src/js/pages/auth/Login.jsx [hunk 1]
```diff
@@ -27,15 +33,25 @@ export default () => {
   return (
     <Simple>
       {inviteLink && (
-        <p>Login to accept invitation {inviteLink}</p>
+          <div className="my-5">
+            <p className="text-secondary">Login to accept invitation {inviteLink}</p>
+          </div>
       )}
+      <div className="my-5">
+          <h2 className="text-dark h4 mb-2">Welcome back!</h2>
```
Comment (dpordomingo) [inline src/js/pages/auth/Login.jsx:41]: It is not expected that the user "came back again", but "came for the very first time".

Comment (eiso) [reply src/js/pages/auth/Login.jsx:41]: Good point, let's make it "Welcome!" and move on.

**Comment by eiso** [comment]
@warenlg shall we merge this?

**Comment by znegrin** [comment]
I wasn't sure if I had to do it, but just updated the copy on the login page:

image

**Comment by warenlg** [comment]
Thanks for the update Zuri, I’m good David

**Pull request merged**